app/src/organisms/ErrorRecoveryFlows/shared/TwoColLwInfoAndDeck.tsx

```diff
@@ -243,10 +243,16 @@ export function TwoColLwInfoAndDeck(
                 )}
                 {labwareRenderInfo
                   .filter(l => l.labwareId !== failedLwId)
-                  .map(({ x, y, labwareDef, labwareId }) => (
-                    <g key={labwareId} transform={`translate(${x},${y})`}>
+                  .map(({ labwareOrigin, labwareDef, labwareId }) => (
+                    <g
+                      key={labwareId}
+                      transform={`translate(${labwareOrigin.x},${labwareOrigin.y})`}
+                    >
                       {labwareDef != null && labwareId !== failedLwId ? (
```

Comment:
```suggestion
                      {labwareDef != null ? (
```
I know you didn't write it, but I think this check is redundant given the `filter` earlier. Is this just for type check reasons?

Reply:
Nope, looks totally vestigial. Will delete.